changelog/4270.feature.rst

```diff
@@ -0,0 +1,2 @@
+Allow `sunpy.visualization.animators.ArrayAnimatorWCS` to disable ticks for
```

Comment:
Can you extend this a bit to say how this is done? Something like "This can be done by setting `ticks=False` in the `coord_params` dictionary."

Reply:
done